x-pack/solutions/security/plugins/security_solution/public/asset_inventory/links.ts

```diff
@@ -8,11 +8,16 @@
 import { i18n } from '@kbn/i18n';
 
 import { INVENTORY } from '../app/translations';
-import { ASSET_INVENTORY_PATH, SecurityPageName, SERVER_APP_ID } from '../../common/constants';
+import {
+  ASSET_INVENTORY_PATH,
+  SecurityPageName,
+  SECURITY_FEATURE_ID,
+} from '../../common/constants';
 import type { LinkItem } from '../common/links/types';
 
 export const links: LinkItem = {
-  capabilities: [`${SERVER_APP_ID}.show`],
+  capabilities: [`${SECURITY_FEATURE_ID}.show`],
+  hideTimeline: true,
```

Comment:
does this remove the timeline from the bottom of the page? i think we're supposed to support it

Reply:
Yes, I'm not sure about supporting it as I couldn't find a mention of it in the Epics or mocks, that's why I have removed it as the Findings page also doesn't include it.

cc @tinnytintin10 - do you have any take on that? should timelines be enabled for the asset inventory page?

Reply:
confirmed with @tinnytintin10 that we intend to keep the timeline